packages/core/aqua-to-js-compiler/src/generate/index.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getPackageJsonContent } from '../utils.js';
+import { FunctionGenerator } from './function.js';
+import { CompilationResult } from '@fluencelabs/aqua-api/aqua-api.js';
+import { JSTypeGenerator, TSTypeGenerator } from './interfaces.js';
+import { ServiceGenerator } from './service.js'; 
+
+type OutputType = 'js' | 'ts';
+
+export default async function ({ services, functions }: CompilationResult, outputType: OutputType) {
+    const typeGenerator = outputType === 'js' ? new JSTypeGenerator() : new TSTypeGenerator();
+    const { version, dependencies } = await getPackageJsonContent();
+    return `/* eslint-disable */
+// @ts-nocheck
+/**
+ *
+ * This file is auto-generated. Do not edit manually: changes may be erased.
+ * Generated by Aqua compiler: https://github.com/fluencelabs/aqua/.
```

Comment:
I don't know if my comment dissapeared somewhere or I didn't post it. anyway.

I just wanna say that these two lines sound repettitive. I would remove them

I think it's enough to say at the top that

```
* This file is generated using:
* @fluencelabs/aqua-api version: ${dependencies['@fluencelabs/aqua-api']}
* @fluencelabs/aqua-to-js-compiler version: ${version}
```
`may be erased` part is redundant in my opinion - it's generated file - and that's all user needs to know.

I don't like `AIR generator version` because it's totally misrepresenting what is this version - when this is the version of @fluencelabs/aqua-api in fact

Reply:
I agree, fixed